shared/chat/inbox/container/index.js

```diff
@@ -37,8 +37,12 @@ const mapStateToProps = state => {
 }
 
 const mapDispatchToProps = dispatch => ({
+  // a hack to have it check for marked as read when we mount as the focus events don't fire always
   _onInitialLoad: (conversationIDKeys: Array<Types.ConversationIDKey>) =>
     dispatch(Chat2Gen.createMetaNeedsUpdating({conversationIDKeys, reason: 'initialTrustedLoad'})),
+  _onMountedDesktop: () => {
```

Comment:
ideally we'd use navigationevents to know when its shown but that seems to not fire consistently. TODO figure that out later. so on inbox mount on desktop we just do an extra check